framework/src/main/java/org/checkerframework/framework/util/dependenttypes/DependentTypesHelper.java

```diff
@@ -566,7 +457,7 @@ public void standardizeExpression(ExpressionTree tree, AnnotatedTypeMirror annot
      * @param type the type to standardize
```

Comment:
It's a bit pedantic, but please note whenever a formal parameter is side-effected by a method.